WalletWasabi/Helpers/LabelBuilder.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace WalletWasabi.Helpers
+{
+	public class LabelBuilder
+	{
+		public List<string> Labels { get; } = new List<string>();
+
+		public LabelBuilder(params string[] labels)
+		{
+			labels = labels ?? new string[] { };
+			foreach (var label in labels)
+			{
+				Add(label);
+			}
+		}
+
+		public void Add(string label)
+		{
+			var parts = Guard.Correct(label).Split(',', StringSplitOptions.RemoveEmptyEntries);
+			foreach (var corrected in parts.Select(Guard.Correct))
+			{
+				if (corrected == "")
```

Comment:
I think this is not possible.

Reply:
Guard.Correct corrects everything to `""`. Whitespaces, nulls. So it should be possible.